js-core/homeworks/phoneApp/src/main.js

```diff
@@ -53,7 +59,7 @@ PhoneApp.prototype.add = function(name, number, surname = null, city = null, com
         id: idNumber,
         name: name,
         surname: surname,
-        number: this.sortNumber(number),
+        number: this.formatedPhoneNumber(number),
```

Comment:
phoneNumbers will be always a string because it's easiest way to handle cases with zeros 
> 099-13-15-100